src/redux/boards/actions.js

```diff
@@ -94,6 +97,34 @@ const actionCreators = {
       type: actions.DELETE_BOARD_FAILURE,
       payload: { error }
     };
+  },
+  addBoard(boardName, boardColor) {
+    return async dispatch => {
+      dispatch({ type: actions.ADD_BOARD });
+      try {
+        const response = await BoardService.addBoard(boardName, boardColor);
+        if (response.ok) {
+          dispatch(actionCreators.addBoardSuccess());
+          dispatch(actionCreators.getBoards());
+        } else {
+          throw new Error(response);
+        }
+      } catch (e) {
+        dispatch(actionCreators.addBoardFailure(e));
+      }
+    };
+  },
+  addBoardSuccess(boardId) {
+    return {
+      type: actions.ADD_BOARD_SUCCESS,
+      payload: { boardId }
+    };
+  },
+  addBoardFailure(error) {
+    return {
+      type: actions.ADD_BOARD_FAILURE,
+      payload: { error }
+    };
```

Comment:
Later we should show that none of these should be exported. Here appears `privateActionCreators`.